### PR TITLE
feat: userRole code field is added

### PR DESCRIPTION
**Pull request by borkarsaish65**

### CODE REVIEW

File: src/models/userRoleExtension.js [hunk 1]
```diff
@@ -34,5 +34,9 @@ module.exports = {
 			type: String,
 			default: 'SYSTEM',
 		},
+		code: {
```
Comment (VISHNUDAS-tunerlabs) [inline src/models/userRoleExtension.js:37]: @borkarsaish65 Add validation to make sure that if code is not provided during the creation of user extension we will throw a validation error

**Review by VISHNUDAS-tunerlabs — CHANGES_REQUESTED** [review]
Reviewed on Oct - 18

File: src/models/userRoleExtension.js (hunk 1 above)
Comment (borkarsaish65) [reply src/models/userRoleExtension.js:37]: done.

**Review by VISHNUDAS-tunerlabs — APPROVED** [review]
Reviewed oct 21

**Pull request merged**